library-extensions/src/main/java/com/mikepenz/fastadapter_extensions/utilities/SubItemUtil.java

```diff
@@ -305,6 +305,54 @@ else if (!subItemsOnly && getParent(item) == null) {
         return deleted;
     }
 
+    /**
+     * notifies sub items, if they are currently extended
+     *
+     * @param adapter the adapter
+     * @param identifiers set of identifiers that should be notified
+     */
+    public static <ITEM extends IItem & IExpandable> void notifyItemsChanged(final FastAdapter adapter, Set<Long> identifiers) {
```

Comment:
As the german comment indicates, this was an accident... I just wanted to check in to my usb stick git repo instead of my github repo...

1) all generics are named T in the `SubItemUtil`, but I change this one to `Item`, although I don't like this if only one generic type is used. But I don't care.
2) I won't change the generic adapter. I could not use this method anymore in my code. My method supports any adapter, with any kind of items, no need to force all items to be `IExpandable` or anything else...